web_notify/static/src/js/services/notification_services.esm.js

```diff
@@ -0,0 +1,53 @@
+import {markup} from "@odoo/owl";
+import {registry} from "@web/core/registry";
+
+export const webNotificationService = {
+    dependencies: ["bus_service", "notification", "action"],
+
+    start(env, {bus_service, notification: notificationService, action}) {
+        function displaywebNotification(notification) {
+            let buttons = [];
+            if (notification.action) {
+                const params =
```

Comment:
```suggestion
                const params = notification.action.context?.params || {};
```
better with `Optional Chaining` was introduced in ES2020 (https://www.geeksforgeeks.org/javascript-optional-chaining/)